release/karaf/itests/camel/src/test/java/org/jboss/integration/fuse/karaf/itest/kiecamel/KieCamelJBPMWorkitemsQuickstartTest.java

```diff
@@ -0,0 +1,129 @@
+        options.add(org.ops4j.pax.exam.CoreOptions.bundle(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.apache.httpcomponents").
+                                                          artifactId("httpclient-osgi")
+                                                          .version("4.3.6").getURL()));
+        options.add(org.ops4j.pax.exam.CoreOptions.bundle(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.apache.httpcomponents").
+                                                          artifactId("httpcore-osgi").
+                                                          version("4.3.3").getURL()));
+        /*options.add(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.ops4j.pax.web.itest").
+                                                          artifactId("pax-web-itest-base")
+        .version("4.2.5"))*/;
+
+        return options.toArray(new Option[1]);
+
+    }
+
+
+
+    @Test(timeout = 100000)
+    public void test() throws Exception {
+
+        // For some reason the War Bundle is not initiated. Check why and remove this comment
+        /* int port = 8181;
+       Bundle[] bundles = bundleContext.getBundles();
+        Bundle warBundle = null;
+        for (Bundle bundle : bundles) {
+            if (bundle.getSymbolicName().equals("org.jboss.integration.fuse.quickstarts.camel.jbpm.workitems")) {
+                warBundle = bundle;
+                break;
+            }
+        }
+        if (warBundle != null) {
+            System.out.println("War Bundle State=" + warBundle.INSTALLED);
+            warBundle.start();
+            System.out.println("War Bundle=" + warBundle.getSymbolicName() + " started");
+            Thread.sleep(10000);
+        }
+        Assert.assertNotNull(warBundle);
+        CloseableHttpClient httpclient = HttpClients.createDefault();
+        HttpGet httpGet = new HttpGet("http://localhost:" + port + "/jbpm-workitems-camel/mortgage");
+        CloseableHttpResponse response1 = httpclient.execute(httpGet);
+        try {
+            System.out.println("RESPONSE-STATUS " + response1.getStatusLine());
+            assertTrue(response1.getStatusLine().getStatusCode() == 200);
+            HttpEntity entity1 = response1.getEntity();
+            // do something useful with the response body
+            // and ensure it is fully consumed
+            StringWriter writer = new StringWriter();
+            IOUtils.copy(entity1.getContent(), writer);
+            String content = writer.toString();
+            System.out.println("RESPONSE:\n" + content);
+            assertTrue(content.contains("Accepted Applications"));
+            assertTrue(content.contains("Rejected Applications"));
+
+        } finally {
+            response1.close();
+        }*/
```

Comment:
What do you think about creating Jira for that and paste the code there? That way we would not loose it and we would avoid commented code in the repo.